tests/Dnsimple/Service/RegistrarTest.php

```diff
@@ -89,6 +89,22 @@ public function testRegisterDomain()
         self::assertEquals("2016-12-09T19:35:31Z", $registration->updatedAt);
     }
 
+    public function testGetDomainRegistration()
+    {
+        $this->mockResponseWith("getDomainRegistration/success");
+        $registration = $this->service->getDomainRegistration(1010, "example.com", 361)->getData();
+
+        self::assertIsInt($registration->id);
```

Comment:
Is there a reason for asserting types instead of the expected values (eg. `self::assertEquals(999, $registration->domainId)`)?

Reply:
Initially, I wanted to have a functional test rather than a regression test as it seemed excessive to have regression tests at every endpoint test, however, after some consideration and the fact that the current test suite does not cover mappings I have reverted to writing regression assertions.